.mvn/develocity.xml

```diff
@@ -22,16 +22,25 @@
 <develocity
         xmlns="https://www.gradle.com/develocity-maven" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
         xsi:schemaLocation="https://www.gradle.com/develocity-maven https://www.gradle.com/schema/develocity-maven.xsd">
+  <projectId>ratis</projectId>
+  <server>
+    <url>https://develocity.apache.org</url>
+    <allowUntrusted>false</allowUntrusted>
+  </server>
   <buildScan>
     <capture>
-      <fileFingerprints>false</fileFingerprints>
-      <buildLogging>false</buildLogging>
-      <testLogging>false</testLogging>
+      <fileFingerprints>true</fileFingerprints>
```

Comment:
These three `true`s for `fileFingerprints`, `buildLogging`, and `testLogging` are all defaults if you'd prefer to just omit them. But no problems explicitly declaring defaults if that is what you prefer to do.

Reply:
@clayburn , thanks for the info.  @adoroszlai , please see which one is more preferable.

Reply:
I prefer explicitly setting these.